lib/stove/community.rb

```diff
@@ -86,9 +86,10 @@ def yank(name)
     #
     def connection
       @connection ||= ChefAPI::Connection.new do |conn|
-        conn.endpoint = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
-        conn.client   = ENV['STOVE_USERNAME'] || Config.username
-        conn.key      = ENV['STOVE_KEY']      || Config.key
+        conn.endpoint   = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
+        conn.client     = ENV['STOVE_USERNAME'] || Config.username
+        conn.key        = ENV['STOVE_KEY']      || Config.key
+        conn.ssl_verify = ENV['SSL_VERIFY'].nil? ? Config.ssl_verify : ENV['SSL_VERIFY']
```

Comment:
ENV['STOVE_SSL_VERIFY'] can be nil, 'true' or 'false' and Config.ssl_verify is true by default, so || doesn't really work in this case